acceptance/bundle/resources/permissions/factcheck/databricks.yml.tmpl

```diff
@@ -0,0 +1,22 @@
+bundle:
+  name: test_bundle_$UNIQUE_NAME
+
+resources:
+  jobs:
+    job:
+      name: job1
+      tasks:
+        - task_key: main
+          notebook_task:
+            notebook_path: /Workspace/Users/user@example.com/notebook
+            source: WORKSPACE
+          new_cluster:
```

Comment:
You can omit this and the `source` field for the notebook task and it will default to serverless.

Makes the config smaller and doesn't impact the test.